draft-ietf-tls-ctls.md

```diff
@@ -255,6 +255,11 @@ to know the minimum safe Finished size. See {{RFC8446}};
 Section E.1 for more on this, as well as
 https://mailarchive.ietf.org/arch/msg/tls/TugB5ddJu3nYg7chcyeIyUqWSbA.]]
 
+optional (object):
+: contains keys that are not required to be understood by the client.
+The server MUST be able to accept connections that do or do not use any of
+these keys.  A key MUST NOT appear in both the main template and the optional
+section.
```

Comment:
Yes, this can only be used for keys that the server can disambiguate.  I'm not necessarily claiming that any such keys have been defined, but I think we might as well agree ahead of time on where to put them, for forwards-compatibility.

My main point here is that all the other keys are _not_ optional, and if you don't recognize one then you can't use the template.

Reply:
OK, that's fair. Could you add some text to the effect about disambiguate and I'll merge thjs?

Reply:
Done

Reply:
Done